lib/ood_core/batch_connect/template.rb

```diff
@@ -68,6 +70,10 @@ def to_s
         EOT
       end
 
+      def passwd_size
```

Comment:
It is unfortunate that the original function was titled `create_passwd` instead of `create_password` but IMHO we shouldn't propagate the abbreviation everywhere. Its preferable to use `password_size` i.e. `def password_size` and `DEFAULT_PASSWORD_SIZE` and `batch_connect.password_size`.